tests/plugins/test_ddp_plugin_with_comm_hook.py

```diff
@@ -108,3 +110,33 @@ def test_ddp_spawn_fp16_compress_comm_hook(tmpdir):
     )
     trainer.fit(model)
     assert trainer.state.finished, f"Training failed with {trainer.state}"
+
+
+@RunIf(skip_windows=True, min_torch="1.9.0", min_gpus=2, special=True)
```

Comment:
```suggestion
@RunIf(skip_windows=True, min_torch="1.10.0", min_gpus=2, special=True)
```
you probably have to put 1.10 right? this hook does not seem to exist in pytorch 1.9

Reply:
Thanks! I think that will explain the lack of definition.

Reply:
The definition of `_TORCH_GREATER_EQUAL_1_10` is also added.